scripts/check_frontend_coverage_test.py

```diff
@@ -224,21 +218,26 @@ def test_check_coverage_changes_decrease(self):
             with fully_covered_tests_swap:
                 with self.assertRaisesRegexp(
                     SystemExit,
-                    r'\033\[1mfile.ts\033\[0m file is in the whitelist but its'
-                    ' coverage decreased. Make sure it is fully covered'
-                    ' by Karma unit tests.\n'):
+                    r'\033\[1mfile.ts\033\[0m seems to be fully covered!'
+                    ' Before removing it manually from the blacklist'
+                    ' in the file'
+                    ' scripts/check_frontend_test_coverage.py, please'
+                    ' make sure you\'ve followed the unit tests rules'
+                    ' correctly on:'
+                    ' https://github.com/oppia/oppia/wiki/Frontend-unit-tests-guide#rules\n'): # pylint: disable=line-too-long
```

Comment:
Avoid using pragma* divide the string into multi-line*

Reply:
Done.